ballerina/vector-store.bal

```diff
@@ -35,27 +35,21 @@ public type VectorStore distinct isolated object {
 
     # Deletes a vector entry from the store by its unique ID.
     #
-    # + id - The unique identifier of the vector entry to delete
+    # + ids - The unique identifiers of the vector entries to delete
     # + return - An `ai:Error` if the operation fails; otherwise, `nil`
-    public isolated function delete(string id) returns Error?;
+    public isolated function delete(string|string[] ids) returns Error?;
```

Comment:
May be we can add `deleteEntry`, `deleteEntries`

Reply:
it will be a breaking change if we remove the existing delete method and introduce new one. for now we'll go ahead with the existing name. WDYT?